css/main.css

```diff
@@ -1,3 +1,769 @@
+    .promo-wrapper-3 {
+        grid-column: 3;
+        text-align: left;
+        padding: 1rem 2rem;
+   }
+   .promo-section > div > p{
+       margin-top: 1rem;
+   }
+
+   .background-color-2-section {
+       background-color: var(--main-color);
+       grid-column: span 12;
+       display: grid;
+       grid-template-columns: repeat(12, 1fr);
+       grid-template-rows: 100px auto auto auto auto 100px;
+       text-align: left;
+       gap: 1rem;
+   }
+   .background-color-2-section-background {
+       grid-column: 1/13;
+       grid-row: 1/6;
+       background-color: var(--background-color-2);
+   }
+   .background-color-2-section-img {
+       grid-column: 2/7;
+       grid-row: 2/7;
+   }
+   .background-color-2-section > h2 {
+       grid-column: 7/13;
+       grid-row: 2;
+       padding: 1rem 0;
+   }
+   .background-color-2-section > p {
+       grid-column: 7/13;
+       grid-row: 3;
+       padding: 2rem 0;
+   }
+   .background-color-2-section > button {
+       margin: 0 0 2rem 0;
+       grid-column: 7/10;
+       grid-row: 5;
+   }
+   .background-color-1-section {
+       gap: 1rem;
+       text-align: left;
+       grid-template-rows: repeat(6, 100px) repeat(2, auto) 80px;
+   }
+   .background-color-1-section > h2 {
+       grid-column: 1/13;
+       grid-row: 1;    
+       padding-top: 3rem;   
+       background-color: var(--main-color);
+   }
+   .background-color-1-section > p {
+       padding: 2rem 0;
+   }
+   .background-color-1-section-background{
+       grid-column: 1/13;
+       grid-row: 1/4;
+       background-color: var(--main-color);
+   }
+   .background-color-1-section-img-1 {
+       grid-column: 1/5;
+       grid-row: 3/6;
+   }
+   .background-color-1-section-img-2 {
+        grid-column: 5/9;
+        grid-row: 2/5;
+   }
+   .background-color-1-section-img-3 {
+       grid-column: 9/13;
+       grid-row: 3/6;
+   }
+   .color-1-section-img-1-header {
+       grid-column: 1/5 ;
+       grid-row: 7;
+   }
+   .color-1-section-img-1-p {
+       grid-column: 1/5;
+       grid-row: 8;
+   }
+   .color-1-section-img-2-header {
+       grid-column: 5/9;
+       grid-row: 7;
+   }
+   .color-1-section-img-2-p {
+       grid-column: 5/9;
+       grid-row: 8;
+   }
+   .color-1-section-img-3-header {
+       grid-column: 9/13;
+       grid-row: 7 ;
+   }
+   .color-1-section-img-3-p {
+       grid-column: 9/13;
+       grid-row: 8;
+   }
+   .background-color-1-section > button{
+        grid-column: 5/9;
+        grid-row: 9;
+   }
+    .background-main-color-section {
+       background-color: var(--main-color);
+       grid-column: span 12;
+       display: grid;
+       grid-template-columns: repeat(12, 1fr);
+       grid-template-rows: repeat(6, auto);
+       text-align: left;
+       gap: 1rem;
+   }
+   .background-main-color-section-img {
+       grid-column: 2/7;
+       grid-row: span 5;
+   }
+   .background-main-color-section > h2 {
+       grid-column: 7/13;
+       padding: 1rem 0;
+   }
```

Comment:
I think the image could start in the first column here (1/6) and the text start from 6 to 13, to better match the design. 
Remember to give vertical breathing space as well (the design has white space at the top of this block). You can do this by adding padding-top and bottom to the whole section (easier than doing this with the grid).